# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: parser/html/nsHtml5Highlighter.h
# Commit: 77388b0795b4
# Full Hash: 77388b0795b4f9a30171c922d6b649c52e5f6c87
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-23 09:34:36
# Description:
#   Bug 1745142 - Communicate encoding commitment via speculative load queue. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D133996
# ==============================================================================

diff -r a388127095db -r 77388b0795b4 parser/html/nsHtml5Highlighter.h
--- a/parser/html/nsHtml5Highlighter.h	Wed Dec 22 15:47:08 2021 +0000
+++ b/parser/html/nsHtml5Highlighter.h	Wed Dec 22 15:54:49 2021 +0000
@@ -47,6 +47,11 @@
   void Start(const nsAutoString& aTitle);
 
   /**
+   * Updates the charset source via the op queue.
+   */
+  void UpdateCharsetSource(nsCharsetSource aCharsetSource);
+
+  /**
    * Report a tokenizer state transition.
    *
    * @param aState the state being transitioned to